components/pipedream/actions/delete-subscription/delete-subscription.mjs

```diff
@@ -0,0 +1,34 @@
+import pipedream from "../../pipedream.app.mjs";
+
+export default {
+  key: "pipedream-delete-subscription",
+  name: "Delete a Subscription",
+  description: "Delete a Subscription. [See Doc](https://pipedream.com/docs/api/rest/#delete-a-subscription)",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    pipedream,
+    emitterId: {
+      propDefinition: [
+        pipedream,
+        "emitterId",
+      ],
+    },
+    eventName: {
+      type: "string",
+      label: "Event name",
+      description: "Event name",
```

Comment:
Is there a more clear description to be used here?
Also, from the docs it looks like this prop should be optional.

Reply:
This is not too clear from the documentation
https://pipedream.com/docs/api/rest/#listen-for-events-from-another-source-or-workflow

Reply:
Yeah. I got a better description